h2o-core/src/main/java/water/parser/FVecParseReader.java

```diff
@@ -32,4 +32,5 @@ public FVecParseReader(Chunk chk){
   @Override public long getGlobalByteOffset(){
     return _goffset;
   }
+  public Chunk getChunk() { return _chk; }
```

Comment:
Fully public? Comment would be helpful since it returns directly underlying chunk.